### PR TITLE
Install MkDocs Documentation 3635

### LINKED ISSUE #3635: Updating Wiki: Install MkDocs
Dependency
- #3634

### Overview
We need to install MkDocs into the new repository so that we can start to move the content from old wiki to new wiki

### Action Items
- [x] research MkDoc requirements
- [x] make a record of the research/requirements and choose a place in the wiki to keep the research 
 - [x] read the documentation on MkDocs for any additional info you might need to install (and update our documentation with the steps/choices)
 - [x] install MkDocs in the repository listed under Resources/Instructions
 - once MkDocs have been installed in the repository 
    - [x] Move issue #3636 to New Issue Approval column and remove the Dependency label
    - [x] post the link to the research/requirements under Resources/Instructions in issue #3636

### Resources/Instructions
- new repo: https://github.com/hackforla/website-wiki
- https://www.mkdocs.org/ 

#### Themes for mkdocs
- [Material for MkDocs](https://squidfunk.github.io/mkdocs-material/getting-started/) is a theme for MkDocs, a static site generator geared towards (technical) project documentation.
- [Jekyll Doc](https://idratherbewriting.com/documentation-theme-jekyll/index.html) is a documentation theme for [Jekyll](https://jekyllrb.com/). 

#### Article comparing MK Docks vs Jekyll for WIKI
* [MkDocs or Jekyll for Technical Documentation](https://blogs.sap.com/2020/07/24/mkdocs-or-jekyll-for-technical-documentation/) - we decided to go with mkdocks

#### Examples
The Youth Justice Nav Project's documentation currently uses Material for MkDocs.
* [Youth Justice Nav repo](https://github.com/hackforla/youthjusticenav-info/tree/main/docs)
* [Youth Justice Nav documentation static site](https://hackforla.github.io/youthjusticenav-info/Juvenile%20Hall/)

GreenearthOS uses Mkdocks without material
- https://www.greenearthos.org/

#### other stuff

- [notes](https://docs.google.com/document/d/1Se4HW4Ab_UR7tgy9Ul8M0Rxev_wrL36rvPPGqYB93uU/edit)
- [audit spreadsheet](https://docs.google.com/spreadsheets/d/1QzfMX3URWfpfeoun34jRm8oDOxXMAMMRvUDS05WLpNY/edit#gid=0)
- Example of another mkdocks install https://brigade.cloud/projects/

### PR DESCRIPTION
**Pull request by t-will-gillis**
Fixes https://github.com/hackforla/website/issues/3635

### What changes did you make and why did you make them ?
- Note that MkDocs is already installed and live
- From discussion with Bonnie in the Sunday meeting, 12 March 23:
  - Added description/ documentation of which files have been installed so far
  - Added list of possible extensions/ plugins/ other options that we may want to consider. These are basic decisions such as adding a format color scheme, TOC, navbars, search capability, etc.

### CODE REVIEW

**Review by JessicaLucindaCheng — APPROVED** [review]
We looked at it during the Sun All Team meeting on 2023-03-19 and Bonnie approved it.

File: README.md [hunk 1]
```diff
@@ -1,73 +1,93 @@
...
+- [Setting up the footer](https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-footer/#setting-up-the-footer)  
+- [Adding a git repository](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#adding-a-git-repository)  
+- [Adding a comment system](https://squidfunk.github.io/mkdocs-material/setup/adding-a-comment-system/#adding-a-comment-system)  
+- [Building an optimized site](https://squidfunk.github.io/mkdocs-material/setup/building-an-optimized-site/#building-an-optimized-site)  
+- [Building for offline usage](https://squidfunk.github.io/mkdocs-material/setup/building-for-offline-usage/#building-for-offline-usage)  
+
+### Python Markdown extension options include:   
+  
+  
+- [Minimal configuration]() when using Material for the first time
```
Comment (JessicaLucindaCheng) [inline README.md:68]: There is no link for this

File: README.md [hunk 2]
```diff
@@ -1,73 +1,93 @@
...
+- [Recommended configuration]() that enables all markdown-related features
```
Comment (JessicaLucindaCheng) [inline README.md:69]: There is no link for this.

File: README.md [hunk 3]
```diff
@@ -1,73 +1,93 @@
...
+  - [InlineHighlight](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#inlinehilite) – built on top of Highlight for highlighting inline code blocks  
+  - [Keys](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#keys) – adds ability to render keyboard keys (e.g. Ctrl + Alt + Del)  
+  - [Markdown in HTML](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#markdown-in-html) – allows wrapping markdown content with custom elements in HTML  
+  - [SmartSymbols](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#smartsymbols) – converts characters into symbols, e.g. fractions, copyright mark, etc.  
+  - [Snippets](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#snippets) – add ability to embed content from arbitrary files into a document  
+  - [SuperFences](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#superfences) – facilitates creation of flowcharts, sequence diagrams, etc.  
+  - [Tabbed]() – allows usage of content tabs
```
Comment (JessicaLucindaCheng) [inline README.md:89]: There is no link for this.

File: README.md [hunk 4]
```diff
@@ -1,73 +1,93 @@
...
+  
+  
+- [Minimal configuration]() when using Material for the first time
+- [Recommended configuration]() that enables all markdown-related features
+- Individual listings of markdown-related config options:
+  - [Abbreviations](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#abbreviations) – ability to add a small tooltip to an element  
+  - [Admonition](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#admonition) – support for call-outs  
+  - [Arithmatex](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#arithmatex) – allows rendering of math typesetting via MathJax  
+  - [Attribute Lists](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#attribute-lists) – allows adding HTML attributes and CSS classes  
+  - [BetterEm](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/#betterem) – improves detection to emphasize text using spec. characters, i.e. bold, italic
```
Comment (JessicaLucindaCheng) [inline README.md:75]: Where it says, `spec. characters`, spell out `spec.` so it's clear what stands for

**Review by JessicaLucindaCheng — COMMENTED** [review]
@t-will-gillis Good job on this documentation! I just noticed some broken links and an abbreviated word that I think should be spelled out for clarity. See my in-line comments.

Sorry for having you make the changes after I merged. The easiest way to make changes would probably be to update your `main` branch on your forked `website-wiki` and make a new branch out of it. Then make the changes on that branch and make a new pull request.

File: README.md (hunk 1 above)
Comment (t-will-gillis) [reply README.md:68]: Please see [hackforla/website-wiki/pull/4](https://github.com/hackforla/website-wiki/pull/4)

File: README.md (hunk 2 above)
Comment (t-will-gillis) [reply README.md:69]: Please see [hackforla/website-wiki/pull/4](https://github.com/hackforla/website-wiki/pull/4)

File: README.md (hunk 3 above)
Comment (t-will-gillis) [reply README.md:89]: Please see [hackforla/website-wiki/pull/4](https://github.com/hackforla/website-wiki/pull/4)

File: README.md (hunk 4 above)
Comment (t-will-gillis) [reply README.md:75]: Please see [hackforla/website-wiki/pull/4](https://github.com/hackforla/website-wiki/pull/4)

**Comment by t-will-gillis** [comment]
Please see https://github.com/hackforla/website-wiki/pull/3

**Comment by JessicaLucindaCheng** [comment]
- The corrections are in #4

**Pull request merged**